tests/unit/scroll/tests.html

```diff
@@ -3475,6 +3478,66 @@
 				start();
 			}, 2500);
 	    });
+		
+		test(testId_126, function () {
+	        $("#testBed").append($('<div id="elemVH" style="height:400px; width: 600px; overflow: hidden;" tabindex="1"></div>').append(contentScrollVH));
```

Comment:
You can cache the newly created element here and reuse it as a cached reference down the test, instead of selecting it by ID. There will also no longer be need to put ID on it.